client/state/action-types.js

```diff
@@ -67,6 +67,15 @@ export const PAGE_LINK_ADD = 'PAGE_LINK_ADD';
 export const PAGE_META_ADD = 'PAGE_META_ADD';
 export const PAGE_TITLE_SET = 'PAGE_TITLE_SET';
 export const PAGE_UNREAD_COUNT_SET = 'PAGE_UNREAD_COUNT_SET';
+export const PLUGIN_SETUP_FETCH_INSTRUCTIONS = 'PLUGIN_SETUP_FETCH_INSTRUCTIONS';
```

Comment:
It reads a bit funny, but our naming convention suffixes the verb. So this might be `PLUGIN_SETUP_INSTRUCTIONS_FETCH` https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#actions